core/domain/skill_domain.py

```diff
@@ -838,14 +838,13 @@ def update_explanation(self, explanation):
         """Updates the explanation of the skill.
 
         Args:
-            explanation: dict. The new explanation of the skill.
+            explanation: SubtitledHtml. The new explanation of the skill.
         """
         old_content_ids = []
         if self.skill_contents.explanation:
             old_content_ids = [self.skill_contents.explanation.content_id]
 
-        self.skill_contents.explanation = (
-            state_domain.SubtitledHtml.from_dict(explanation))
+        self.skill_contents.explanation = (explanation)
```

Comment:
Remove the brackets around 'explanation', those aren't required, right?

Reply:
I'm not sure why they were present previously, I'll remove it.

Reply:
This is not resolved.

Reply:
Had locally resolved it. Pushed it.